model/qti/interaction/PortableCustomInteraction.php

```diff
@@ -215,14 +215,17 @@ private function extractPciProperties(DOMElement $propertiesNode, $ns = ''){
      * @param string $ns
      * @return string
      */
-    private function serializePciProperties($properties, $ns = ''){
+    private function serializePciProperties($properties, $ns = '', $name = ''){
         
         $ns = $this->formatPciNs($ns);
-       
-        $returnValue = '<'.$ns.'properties>';
+        if (!empty($name)) {
+            $returnValue = '<'.$ns.'properties key="' . $name . '">';
+        } else {
+            $returnValue = '<'.$ns.'properties>';
+        }
```

Comment:
During importing PCI interaction `key` attribute of `<properties>` tag is missed.
before importing:
```
<pci:properties key="0">
    <pci:entry key="label">Choice 1</pci:entry>
</pci:properties>
```
after importing:
```
<pci:properties>
    <pci:entry key="label">Choice 1</pci:entry>
</pci:properties>
```